src/fides/api/alembic/migrations/versions/c9a22b284afa_add_datahub_connection_type.py

```diff
@@ -0,0 +1,106 @@
+"""Add Datahub connection type
+
+Revision ID: c9a22b284afa
+Revises: 4ebe0766021b
+Create Date: 2024-10-21 13:40:31.313754
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "c9a22b284afa"
+down_revision = "1c8dfd6a1bc6"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add 'datahub' to ConnectionType enum
+    op.execute("ALTER TYPE connectiontype RENAME TO connectiontype_old")
```

Comment:
Did you get any answer from other people about why we do the connectiontype_old instead of altering the existing one? Or just better to keep with how it was down before?

Reply:
tbh I never asked 🙈 I've just seen it done this way always. I'll ask around in the eng channel

Reply:
I think "it's just been done this way before" is a 100% valid argument for doing it in the PR that way, for all we know it's catastrophic with the way we do Alembic and we realized in the past or something. But we should also just get an answer if anyone knows whether that's the case or not. I think good to merge as is and ask.